1080i/Includes_Images.xml

```diff
@@ -329,6 +329,9 @@
     </variable>
 
     <variable name="ChannelListEPGIconVar">
+        <value condition="!String.IsEmpty(Window(Home).Property(PVR.Artwork.fanart)) + Skin.HasSetting(Skin_enablePvrArtwork_Fanart_Channel_Manager) + !Skin.HasSetting(Skin_enablePvrArtwork_Poster_Channel_Manager)+ String.StartsWith(ListItem.FolderPath,pvr://)">$INFO[Window(Home).Property(PVR.Artwork.fanart)]</value>
+        <value condition="!String.IsEmpty(Window(Home).Property(PVR.Artwork.poster)) + !Skin.HasSetting(Skin_enablePvrArtwork_Fanart_Channel_Manager) + Skin.HasSetting(Skin_enablePvrArtwork_Poster_Channel_Manager)+ String.StartsWith(ListItem.FolderPath,pvr://)">$INFO[Window(Home).Property(PVR.Artwork.poster)]</value>
+        <value condition="String.IsEmpty(Window(Home).Property(PVR.Artwork.fanart)) + !String.IsEmpty(Window(Home).Property(PVR.Artwork.poster)) + Skin.HasSetting(Skin_enablePvrArtwork_PosterFallback_Channel_Manager)+ String.StartsWith(ListItem.FolderPath,pvr://)">$INFO[Window(Home).Property(PVR.Artwork.poster)]</value>
```

Comment:
no need to check for
>> String.IsEmpty(Window(Home).Property(PVR.Artwork.fanart))

Reply:
in case you are referring to row 334: This is the fallback scenario which can be activated in the settings to use PvrArtwork.Poster in case PVRArtwork.Fanart is empty. I can remove the String.IsEmpty(Window(Home).Property(PVR.Artwork.fanart)) check if you like. Background: even if fallback is set, only do it if fanart is empty.

Reply:
The fanart condition is true in line 332 if fanart is available and the setting enabled. If there is no fanart available, the next condition will be checked. Then at line 334 there is imho no need to check, if fanart is available, cause is fanart is available, line 332 condition will be true...

But in general, i think that's too many options for this skin. My personal opinion is that there should be only one option (e.g. use poster as fallback), because the PVR area is not designed for posters, but for fanart/thumb.

Reply:
Agreed .. I changed it in a way that PVR Artwork can be enabled per screen (right now Channel Manager only). Once its activated, it is trying fanart first, then poster, then EPGIcon and fallback is Icon.

Please check from your side.

Reply:
Looks good from my side. Why not using it also in the EPG?

Reply:
I'd start with that (and also DialogPVRInfo, OSD etc) after the initial support and first release.